dockerfiles/jupyter/env

```diff
@@ -0,0 +1,18 @@
+# default env vars, will be loaded once MLRun imports/starts
```

Comment:
The standard seems to be to name the file `.env`

Reply:
i think it wont be visible in Jupy w `.` , saw Azure demo it was `env`